homeassistant/components/hue/sensor_base.py

```diff
@@ -277,7 +277,13 @@ class GenericZLLSensor(GenericHueSensor):
 
     @property
     def device_state_attributes(self):
-        """Return the device state attributes."""
-        return {
-            "battery_level": self.sensor.battery
-        }
+        """Return the device state attributes."""                        
+        attributes = dict()                                              
+        attributes.update(self.sensor.config)                            
+        """ Removing unnecessary values."""                              
+        if 'pending' in attributes: del attributes['pending']            
+        if 'reachable' in attributes: del attributes['reachable']        
```

Comment:
We should not add random stuff because it exists. this one for example is represented in available property.